File: 프로그래머스/src/프로그래머스_02252022/음양더하기.cpp
#include <string>
#include <vector>

using namespace std;

/*true : 양수, false : 음수*/
int solution(vector<int> absolutes, vector<bool> signs)
{
    int answer = 0;
    for (int i = 0; i < signs.size(); i++)
    {
        if (signs[i])
            answer += absolutes[i];
        else
            answer -= absolutes[i];
    }

    return answer;
}

File: 2026/Level1/음양더하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/76501?language=python3
def solution(absolutes, signs):
    answer = 0
    current = 0
    for x in range(len(absolutes)):
        current = absolutes[x]
        if signs[x] == False:
            current *= -1
        answer += current
    return answer